css/index.css

```diff
@@ -1,3 +1,354 @@
+  margin: auto;
+  padding-bottom: 100px;
+}
+.footer-content {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+}
+
+.footer-content a {
+  text-decoration: none;
+  color: black;
+  font-weight: 500;
+}
+.copy {
+  text-align: center;
+  padding: 20px;
+}
+
+/* services.html styles */
+
+/* main header */
+.header-content {
+  width: 80%;
+  margin: 50px auto;
+  display: flex;
+  flex-direction: column;
+}
+.header-content h1 {
+  margin-top: 20px;
+  font-size: 1.4rem;
+  font-weight: 500;
+}
+.header-content img {
+  object-fit: cover;
+}
+.header-content p {
+  font-weight: 400;
+  color: black;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+
+/* services */
+.services-content {
+  width: 80%;
+  margin: auto;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.services-content div h1 {
+  font-size: 1.2rem;
+}
+.services-content div a {
+  background: white;
+  display: block;
+  margin-top: 20px;
+  width: 120px;
+  text-align: center;
+  padding: 10px;
+  color: black;
+  text-decoration: none;
+  border: 1px solid black;
+}
+.services-content div {
+  background: lightgray;
+  width: 40%;
+  border: 1px solid black;
+  margin: 20px;
+  padding: 20px;
+}
+
+/* services facts */
+.facts-wrapper {
+  width: 80%;
+  margin: 50px auto;
+  display: flex;
+  align-items: flex-start;
+}
+.facts-wrapper h1 {
+  font-size: 1.3rem;
+}
+.facts-wrapper ul {
+  margin-top: 40px;
+  list-style: square;
+  margin-left: 20px;
+}
+.facts-wrapper li {
+  margin: 20px 0;
+}
+.facts-wrapper img {
+  margin-left: 20px;
+}
+
+/* services.html media queries */
+@media (max-width: 800px) {
+  .header-content,
+  .services-content,
+  .facts-wrapper,
+  hr {
+    width: 90%;
+  }
+  .header-content,
+  .services-content,
+  .facts-wrapper {
+    flex-direction: column;
+  }
+  .services-content div {
+    width: 100%;
+    margin: 20px auto;
+  }
+  .facts-wrapper {
+    justify-content: center;
+    align-items: center;
+  }
+  .main-nav-content {
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+  }
+  .facts-wrapper img {
+    width: 100%;
+    margin: 0;
+  }
+}
+
+/* index.html media queries */
+@media (max-width: 800px) {
+  .main-header,
+  .background-image,
+  footer {
+    width: 90%;
+    flex-direction: column;
+  }
+  .main-content {
+    width: 90%;
+  }
+
+  /* change logo order */
+  .main-nav-content .logo-link {
+    order: -1;
+    align-self: flex-start;
+    flex-direction: column;
+    width: 100%;
+    margin: auto;
+    text-align: center;
+  }
+
+  .main-header img {
+    display: none;
+  }
+  .main-content-about-right,
+  .main-content-about-left {
+    margin-bottom: 20px;
+  }
+  /* .main-content-about-left {
+    margin-top: 40px;
+  } */
+  .background-image {
+    margin: 60px auto;
+  }
+  .services {
+    width: 90%;
+  }
+  hr {
+    margin: 70px auto;
+  }
+}
+
+/* index.html media 500px */
+
+@media (max-width: 500px) {
```

Comment:
best practice would have the media queries be separate from the desktop CSS for readability. In desktop-first it owuld be desktop at the top of the page, then all the tablet-view css, then the phone-view.